w3-store.md

```diff
@@ -143,6 +145,66 @@ If `status == 'upload'`, the response will include additional fields containing
 
 The client should then make an HTTP `PUT` request to the `url` specified in the response, attaching all the included `headers`. The body of the request MUST be CAR data, whose size exactly equals the size specified in the `store/add` invocation's `size` caveat. Additionally, the CID of the uploaded CAR must match the invocation's `link` caveat. In other words, attempting to upload any data other than that authorized by the `store/add` invocation will fail.
 
+### `store/get`
+
+> Get metadata about a CAR shard from a space
+
+The `store/get` capability can be invoked to get metadata about a CAR from a [space](#spaces).
+
+This may be used to check for inclusion, or to get the `size` and `origin` properties for a shard.
+
+#### Derivations
+
+`store/get` can be derived from a [`store/*`](#store) or [`*`][ucan-spec-top] capability with a matching `with` field.
+
+#### Caveats
+
+When invoking `store/get`, the `link` caveat must be set to the CID of the CAR file to get metadata for.
+
+If a delegation contains a `link` caveat, an invocation derived from it must have the same CAR CID in its `link` field. A delegation without a `link` caveat may be invoked with any `link` value.
+
+| `field`   | `value`                           | `required?` | `context`                                     |
+| --------- | --------------------------------- | ----------- | --------------------------------------------- |
+| `link`    | CAR CID string, e.g. `bag...`     |             | The CID of the CAR to get metadata for        |
+
+#### Invocation
+
+```js
+{
+  can: "store/get",
+  with: "did:key:abc...",
+  nb: {
+    link: "bag...",
+  }
+}
+```
+
+#### Responses
+
+*Note*: This section is non-normative and subject to change, pending the [formalization of receipts and invocations][invocation-spec-pr].
+
+Executing a `store/get` invocation with a service provider should return a response object.
+
+If a failure occurs, the response will have an `error` field with a value of `true`, and a `message` string field with details about the error.
```

Comment:
maybe I'm misunderstanding this, but won't the error response look like:

```js
{ error: { name: "ErrorName", message: "An error message" } }
```

?

Reply:
ah yes you right. I was blending this in with the rest of this spec, but things have changed!

I can't find a place in our other specs where we document the expected output shapes which seems odd too.

Reply:
yeaaaa I agree, I think it's a hole in our specs that we haven't documented the input/return types across the board, maybe even a DX issue worth fixing asap?